docs/use/db_provision.md

```diff
@@ -40,7 +40,31 @@ To provision a new database:
 
 5. On the **Resources** page, select the **Number of nodes**. Also, set the resources per node by selecting one of the predefined presets or by specifying the CPU, Memory, and Disk. For more information on resources, see the [Scale database deployment](../use/scaling.md) section.
 
-6. On the **Scheduled Backups** page, set up a schedule if you wish to run backup jobs for your new database.
+    Additionally, based on the database technology you’re working with, select the following:
+
+    - MySQL: On the **Proxies** panel, select the **Number of proxies** and **Resource size per proxy**.
+
+        !!! info
+            Proxies are used primarily to ensure high availability, load balancing, and database failover management. They act as intermediaries, ensuring client requests are directed to the appropriate database instances.
+
+        ![!image](../images/resources_proxies_page.png)
+
+    - MongoDB: On the **Routers** panel, select the **Number of routers** and **Resource size per router**.
+
+        !!! info
+            The query router directs the client queries to the proper shard(s).
+
+        ![!image](../images/resources_routers.png)
+
```

Comment:
We have a bug that's showing this panel all the time but it should only show it if sharding has been enabled. Maybe we should remove this :slightly_smiling_face: 
```suggestion
```